client/network/src/protocol/sync.rs

```diff
@@ -357,6 +360,11 @@ impl<B: BlockT> ChainSync<B> {
 		self.fork_targets.len()
 	}
 
+	/// Number of processed blocks.
+	pub fn num_processed_blocks(&self) -> usize {
```

Comment:
We only need this in tests? :thinking:

Reply:
Yes, unfortunately there's no way to access this private data since the tests are in a separate crate.